tests/test_flow/conftest.py

```diff
@@ -163,7 +177,7 @@ def gcs_url_stem(request, use_fake_gcp):
 
 
 @pytest.fixture
-def session_tmp_gcs_url_prefix(gcs_url_stem, use_fake_gcp):
+def session_tmp_gcs_url_prefix(gcs_url_stem, gcs_fs):
     """
     Sets up and tears down a temporary "directory" on GCS to be shared by all
```

Comment:
Yeah the scope was changed so that it supports both fake and real GCS.

Reply:
Does this mean we do the setup/teardown for each test? Is that expensive?

Also, I'm not following why the scope needs to change to support both real and fake GCS. What's the problem with using session scope? Since `use_fake_gcp` is parameterized, won't pytest still create two separate instances of this fixture?

Reply:
Modified this fixture to support real gcs only. This fixture now runs only if the `--bucket` is passed.